src/IssueViz/TableControls/VSTableEventProcessorProvider.cs

```diff
@@ -76,17 +76,17 @@ void ITableControlEventProcessor.PostprocessSelectionChanged(TableSelectionChang
             {
                 try
                 {
-                    IAnalysisIssue selectedIssue = null;
+                    IAnalysisIssueVisualization selected = null;
```

Comment:
We're now storing the issue viz in the Error List so we can fetch it directly, rather than fetching an issue and converting it separately.